www/profile/app-profile.less

```diff
@@ -79,11 +82,19 @@
             margin: 5px !important;
         }
     }
+
     .cp-app-profile-resizer {
         text-align: center;
     }
-    #cp-app-profile-displayname, #cp-app-profile-link {
-        margin: 10px 0;
+    .cp-app-profile-displayname {
+        margin-top: 1rem;
+    }
+    div.cp-app-profile-link.hidden{
```

Comment:
As mentioned in previous PRs, `hidden` is not a good class name as it could be used by libraries we embed in CryptPad and the style could leak to or from this library.